other/usr/share/locale/en_GB/LC_MESSAGES/kb-gu.translation

```diff
@@ -1 +1 @@
-Gujarati	Gudźarati
```

Comment:
To była oficjalna nazwa: https://pl.wikipedia.org/wiki/J%C4%99zyk_gud%C5%BAarati

Reply:
OK.